packages/datetime/test/common/dateTestUtils.ts

```diff
@@ -6,7 +6,9 @@
  */
 
 import { assert } from "chai";
+import { ReactWrapper } from "enzyme";
 import { padWithZeroes } from "../../src/common/utils";
+import { Classes } from "../../src/index";
```

Comment:
fix import path to actual path to `classes.ts`

really annoying that VS Code chooses _this_ as the automatic path.